ContentPatcher/docs/i18n/zh/author-guide/config.md

```diff
@@ -0,0 +1,180 @@
+﻿← [模组作者指南](../author-guide.md)
+
+设置选项功能让你向玩家提供可更改的设置，并基于设置实现动态。
+
+**🌐 其他语言： [en (English)](../../../author-guide/config.md)。**
+
+## 目录
+* [基本设置](#basic-config)
+  * [概述](#overview)
+  * [设置定义](#define-your-config)
+  * [示例](#examples)
+* [设置菜单](#config-ui)
+  * [显示选项](#display-options)
+  * [分段](#sections)
+  * [翻译](#translations)
+* [参见](#see-also)
+
+## 基本设置<a name="basic-config"></a>
+## 概述<a name="overview"></a>
+
+你可以使用`ConfigSchema`字段定义内容包的设置选项。Content Patcher将自动添加`config.json`文件和[游戏内设置菜单](#config-ui)并允许玩家更改你提供的设置。
+
+在内容包内你可以把设置选项当作[令牌和条件](#../author-guide.md#tokens)使用，从而实现动态改变。
+
+### 设置定义<a name="define-your-config"></a>
+
+使用设置选项的第一步用`ConfigSchema`字段来描述你的内容包所提供的选项。`ConfigSchema`是与`Format`和`Changes`同级的字段。每一个设置选项有一个作为令牌的键，和一个包含一下字段的模型：
+
+一个以逗号分隔的字符串，代表玩家可选的值。如果省略，则允许任何值。
+
+类型                 | 作用
+------------------- | -------
+`AllowValues`       | _（可选）_ 一个以逗号分隔的字符串，代表玩家可选的值。如果省略，则允许任何值。<br />**Tip:** 当你用`"true, false"`定义可启用/禁用的选项时，Content Patcher会认出它是布尔(并在[设置菜单](#config-ui)以复选框显示此设置).
+`AllowBlank`        | _（可选）_ 该字段是否可以留空。如果false或省略，则将用默认值（`Default`）替换空白字段。
+`AllowMultiple`     | _（可选）_ 玩家是否可以指定多个以逗号分隔的值。默认false。
+`Default`           | _(可选，除非`AllowBlank`为false)_ 此设置的默认值。如果`AllowMultiple`为true的，则可以包含多个以逗号分隔的值。如果省略，默认值为空白。
+
+设置选项的名称和字段不区分大小写。
+
+### 示例<a name="examples"></a>
+
+此`content.json`定义了一个名为`BillboardMaterial`的设置，并使用此设置令牌控制补丁效果。
+
+```js
+{
+   "Format": "2.7.0",
+   "ConfigSchema": {
+      "Material": {
```

Comment:
Chinese characters are valid as token names; should we translate the example token names & values?